python/lsst/rubintv/production/processingControl.py

```diff
@@ -668,6 +668,10 @@ def dispatchGatherSteps(self, who: str) -> bool:
             if who == "SFM":
                 (expRecord,) = self.butler.registry.queryDimensionRecords("exposure", dataId=dataCoords[0])
                 self.dispatchOneOffProcessing(expRecord, PodFlavor.ONE_OFF_CALEXP_WORKER)
+            if who == "AOS":
+                for dataCoord in dataCoords:
+                    (expRecord,) = self.butler.registry.queryDimensionRecords("exposure", dataId=dataCoord)
+                    self.dispatchOneOffProcessing(expRecord, PodFlavor.ONE_OFF_POSTISR_WORKER)
```

Comment:
This is doing much more than just calculating the exposure timing, isn't it?

Reply:
It is, yeah. That should have been being done to those exposures all along really, and I hadn't thought about out how to do it, but realised it was actually pretty easy 🙂 You were just checking your understanding and that I was doing what I intended, right? Or were you thinking the PR should maybe be renamed because of this?